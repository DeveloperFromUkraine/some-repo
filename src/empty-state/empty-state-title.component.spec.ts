import { ComponentFixture, TestBed } from "@angular/core/testing";
import { EmptyStateTitleComponent } from "../index";
import { ComponentTest } from "../../test/test-bed/component";

describe('Empty State Title', () => {
    let fixture: ComponentFixture<EmptyStateTitleComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [EmptyStateTitleComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(EmptyStateTitleComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});