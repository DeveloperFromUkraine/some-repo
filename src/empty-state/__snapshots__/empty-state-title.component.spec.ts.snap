// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty State Title should match snapshot 1`] = `
<ign-empty-state-title>
  
</ign-empty-state-title>
`;
